src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write your code here
+function handleSuccess() {
+  const ressolveDiv = document.createElement('div');
```

Comment:
There is a typo in the variable name `ressolveDiv`. It should be `resolveDiv` to match the intended purpose of the variable.